lib/matplotlib/mathtext.py

```diff
@@ -464,8 +454,9 @@ def set_canvas_size(self, w, h, d):
         Set the size of the buffer used to render the math expression.
         Only really necessary for the bitmap backends.
         """
-        self.width, self.height, self.depth = ceil(w), ceil(h), ceil(d)
-        self.mathtext_backend.set_canvas_size(self.width, self.height, self.depth)
+        self.width, self.height, self.depth = map(np.ceil, [w, h, d])
```

Comment:
Is this faster than `np.ceil([w, h, d])`?

Reply:
missed that, thanks.